presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -885,4 +886,16 @@ public FeaturesConfig setLegacyUnnestArrayRows(boolean legacyUnnestArrayRows)
         this.legacyUnnestArrayRows = legacyUnnestArrayRows;
         return this;
     }
+
+    @Config("optimizer.default-filter-factor-enabled")
+    public FeaturesConfig setDefaultFilterFactorEnabled(boolean defaultFilterFactorEnabled)
```

Comment:
move the setter/getter next to other CBO setter/getters (perhaps just after  `setEnableStatsCalculator`)